openedx/adg/lms/registration_extension/tests/test_helpers.py

```diff
@@ -0,0 +1,30 @@
+"""
+All tests for registration_extension helpers functions
+"""
+import pytest
+
+from common.djangoapps.student.tests.factories import UserFactory
+from openedx.adg.lms.registration_extension.constants import RETIRED_EMAIL_MESSAGE
+from openedx.adg.lms.registration_extension.helpers import validate_email_exists_or_retired
+from openedx.core.djangoapps.user_api import accounts
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'is_retired', [True, False],
+)
```

Comment:
Let's add a test for already existing email as well

Reply:
Already testing it.